tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -2082,6 +1818,43 @@ public function userGetsDetailsOfTheGroupUsingTheGraphApi(string $user, string $
 		);
 	}
 
+	/**
+	 * @Then /^the JSON data of the response should (not )?contain (?:display|group) name "([^"]*)"(?: and match)?$/
+	 *
+	 * @param string $shouldOrNot (not|)
+	 * @param string $displayName
+	 * @param PyStringNode|null $schemaString
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function theJsonDataResponseShouldContainDisplayNameAndMatch(
+		string $shouldOrNot,
+		string $displayName,
+		?PyStringNode $schemaString = null
+	): void {
+		if (isset($this->featureContext->getJsonDecodedResponseBodyContent()->value)) {
+			$responseBody = $this->featureContext->getJsonDecodedResponseBodyContent()->value;
+		} else {
+			$responseBody = $this->featureContext->getJsonDecodedResponseBodyContent();
```

Comment:
When we are expecting only one response, we should use the simpler `the JSON data of the response should match` step. This step here should only be used when we have a response with multiple objects where we cannot ensure the order.